.github/workflows/security_dismissal.yml

```diff
@@ -0,0 +1,136 @@
+name: Track Dismissed Code Scanning Alerts
+
+on:
+  push:
+    branches: 
+     - dismissalCheckout   # Trigger only test branch, remember to remove before PRing to grass
+                           # in the grass repo this will be changed to a scheduled event (e.g., every 24 hours)
+
+jobs:
+    get-dismissals-data:
+        runs-on: ubuntu-latest  
+        permissions: write-all
```

Comment:
Find the necessary scopes. There was an action from GitHub security's team that made a local proxy to help identify what scopes were used, if you need help. It dates like 2-3 years ago, I can't find it but used it in the last month.

Reply:
do you remember what file/files n the grass repo it was in? It would helpful to have an example

Reply:
https://github.com/GitHubSecurityLab/actions-permissions
It was that long ago finally.
But it may be faster to read the documentation, and test out with a fork of your fork (I had to make an organisation, echoix-org, to be able to almost-properly test out these kinds of permissions, when I worked on the posting of code suggestions on PRs).

Reply:
That was the blog post that I first learned about it https://github.blog/2023-08-09-four-tips-to-keep-your-github-actions-workflows-secure/